Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := $(shell git describe --tags --dirty=-modified)
+VERSION := $(shell git describe --tags --dirty=-modified --always)
```

Comment:
@nicob87 I have added this extra flag here, as I was constantly getting
errors in this particular line. Please see if you're ok with it.